tests/Microsoft.Identity.Test.Unit/Microsoft.Identity.Test.Unit.csproj

```diff
@@ -4,8 +4,9 @@
     <TargetFrameworkNetDesktop461>net48</TargetFrameworkNetDesktop461>
     <TargetFrameworkNetCore>netcoreapp3.1</TargetFrameworkNetCore>
     <TargetFrameworkNet6Win>net6.0-windows10.0.17763.0</TargetFrameworkNet6Win>
+    <TargetFrameworkNet7Win>net7.0-windows10.0.18362.0</TargetFrameworkNet7Win>
     
-    <TargetFrameworks>$(TargetFrameworkNetDesktop461);$(TargetFrameworkNetCore);$(TargetFrameworkNet6Win);</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworkNetDesktop461);$(TargetFrameworkNetCore);$(TargetFrameworkNet6Win);$(TargetFrameworkNet7Win)</TargetFrameworks>
```

Comment:
How much time does this add to the build? Is it worth removing: 

- the net6 target?
- the netcore 3.1 target ? 

to mitigate ?

Reply:
Unit tests run is about 1.5 minutes. I'm fine with adding additional frameworks, as long as it adds coverage. Net7 broke stuff, but it's also a new platform. net3.1 and net6 should really only get patch changes, so maybe worth replacing at least net6. Netcore3.1 I think we still need, since it tests the Newtonsoft json related code.

More OCE work: maybe we should add test coverage tools or something for each PR run.